CMakeLists.txt

```diff
@@ -33,7 +34,9 @@ if (NOT WIN32)
     find_package(Fluidsynth)
     find_package(Freetype)
     find_package(Fontconfig)
-    find_package(ICU OPTIONAL_COMPONENTS i18n lx)
+    if (NOT APPLE)
+      find_package(ICU OPTIONAL_COMPONENTS i18n lx)
```

Comment:
also, 4 spaces for indentation (also below)

Reply:
Fixed.